assets/dev/scss/frontend/_container.scss

```diff
@@ -19,10 +19,10 @@
 	--min-height: initial;
 	--height: auto;
 	--text-align: initial;
-	--margin-block-start: 0;
-	--margin-inline-end: 0;
-	--margin-block-end: 0;
-	--margin-inline-start: 0;
+	--margin-block-start: 0px;
```

Comment:
'0' dimension values causes problems is not compatible with the css calc() function.

Reply:
Nice Catch